apps/odyssey-stats/src/lib/can-current-user.ts

```diff
@@ -1,5 +1,5 @@
 import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
 import config from './config-api';
 
-export default ( siteId, capability ) =>
+export default ( siteId: number, capability: string ) =>
 	canCurrentUser( config( 'intial_state' ) ?? {}, siteId, capability );
```

Comment:
I just noticed a typo: `intial_state` should be `initial_state`.

Reply:
I fixed this by allowing both `intial_state` and `initial_state`

452acd2647

Reply:
I ended up only adding a comment, and will look later.